set.mm

```diff
@@ -529337,12 +529336,12 @@ at least two objects (see ~ bj-dtru ).  (Contributed by BJ,
     bj-axrep2 $p |- E. x ( E. y A. z ( ph -> z = y ) ->
```

Comment:
What is the reason this does not replace axrep2?

Reply:
Good catch. Actually `bj-axrep1` `bj-axrep2` `bj-axrep3` `bj-axrep4` `bj-axrep5` `bj-axsep` drop ax-13 without adding any $d condition, so they can simply replace the current `axrep1` `axrep2` `axrep3` `axrep4` `axrep5` `axsep` in main. I think I'm going take them, I only have to keep an eye on proof minimizations and appropriate credits.

Reply:
Maybe ax-sep was seen as the first in the order, and the rest follows up.   This is just a wild guess, though.

Reply:
I will also mention [sn-axsep](https://us.metamath.org/mpeuni/sn-axsep.html), incidentally I think it has an extra dv condition on `ph z` (so sn-axsep is actually axsep2)

Reply:
Thanks @wlammen and @GinoGiotto. The main reason these were in my mathbox is that I had done then in a batch and Norm wisely insisted that they be moved to Main only carefully and progressively, to allow careful strutiny, not in a big batch. I have since been busy with other things and didn't carry out the move. So thank you Gino for doing it.

Maybe it's better to merge this one as is, and do what you propose in another MR, once we see the resulting website ?

Reply:
> Maybe it's better to merge this one as is, and do what you propose in another MR, once we see the resulting website ?

Yes, maybe it's a little easier to do two separate PRs so that I put things better in order in my mind.

Reply:
maybe call sn-axrep, axrep1